src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/annotations/AErrorPatternIgnored.java

```diff
@@ -0,0 +1,36 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.integTests.framework.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation can be used to ignore certain log messages for specific test methods.
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@Repeatable(AErrorPatternIgnored.Patterns.class)
+public @interface AErrorPatternIgnored {
+  /**
+   * The substring, which if found in the error log line, we will ignore the error log.
+   *
+   * @return Patterns that will be considered before the error log. If multiple patterns are specified, then the presence of any of the
+   *     them will ignore the log line.
+   */
+  String pattern();
```

Comment:
can we also expose a method to suppress errors at file level granularity ? i.e. we can ignore all errors from some certain RCAs